### PR TITLE
Issue/659/point destription and success message

### PR DESCRIPTION
**Pull request by fzemi**

### CODE REVIEW

File: server/bin/mongodb/default-dump/points.dump.js [hunk 1]
```diff
@@ -175,6 +193,8 @@ const point10 = {
     longitude: 18.543,
     latitude: 54.479,
   },
+  pointDescription: null,
+  pointSuccessMessage: 'Lorem ipsum dolor sit amet',
```
Comment (dbetka) [inline server/bin/mongodb/default-dump/points.dump.js:197]: Null for timeout points

**Review by dbetka — CHANGES_REQUESTED** [review]
To fix

File: server/bin/mongodb/default-dump/points.dump.js (hunk 1 above)
Comment (dbetka) [reply server/bin/mongodb/default-dump/points.dump.js:197]: @fzemi

Comment (fzemi) [reply server/bin/mongodb/default-dump/points.dump.js]: Done

**Review by HenoJurek — APPROVED** [review]
Looks good to me

**Review by HenoJurek — CHANGES_REQUESTED** [review]
Ops, i forgot about swagger docu. Please add documentation

**Review by HenoJurek — APPROVED** [review]

**Review by dbetka — APPROVED** [review]

**Pull request merged**